tests/unittests/Foundation/ReferenceFoundation/TestNSURLResponse.mm

```diff
@@ -34,6 +34,7 @@
     NSURLResponse* res =
         [[[NSURLResponse alloc] initWithURL:url MIMEType:@"text/plain" expectedContentLength:0 textEncodingName:nil] autorelease];
     ASSERT_OBJCEQ_MSG(url, res.URL, @"should be the expected url");
+    ASSERT_OBJCEQ(@"path.txt", res.suggestedFilename);
 }
```

Comment:
These were run on reference platform and they pass.
Yep we should keep the separate, but it was soo tempting to add an extra check here.

---
In reply to: [95276334](https://github.com/Microsoft/WinObjC/pull/1662#discussion_r95276334) [](ancestors = 95276334)